orderbook/openapi.yml

```diff
@@ -87,6 +87,12 @@ paths:
           schema:
             type: boolean
             default: false
+        - name: includeDeniedTokens
```

Comment:
Sorry for being nitpicky here on the naming. Can we call it UnsupportedTokenList? This way it seems more like a technical limitation (which it is) rather than something we do arbitrarily.